qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -601,11 +605,9 @@ def test_issue_cephfs_shell_cmd_at_invocation(self):
         dirname = 'somedirectory'
         self.run_cephfs_shell_cmd(['mkdir', dirname])
 
-        # TODO: Once cephfs-shell can pickup its config variables from
-        # ceph.conf, set colors Never there and get rid of the same in
-        # following comamnd.
+        self.ceph_cluster.set_config_opt('client', 'colors', 'Never')
```

Comment:
I would suggest that in `TestCephFSShell.setUpClass`(https://docs.python.org/2/library/unittest.html#unittest.TestCase.setUpClass) that you set this config so that it affects all test cases. Then, you also need to remember to undo the config change in `tearDownClass`. You would do `config rm client colors`.

Reply:
Done.

Reply:
I guess you just removed this config? I don't see the change.

Reply:
That's odd, I remember writing it. Maybe it got lost somewhere in some push.